package.json

```diff
@@ -411,7 +413,7 @@
   "devDependencies": {
     "all-contributors-cli": "^5.4.0",
     "coveralls": "^2.11.4",
-    "husky": "^0.14.3",
+    "husky": "1.1.1",
```

Comment:
Thank you @cmygray for this PR

I'm afraid we also need to move this line
https://github.com/deerawan/vscode-dash/blob/051c3a3fdbdc1a720688ed9de37a50252132bf44/package.json#L406

to conform with husky 1.1 standard for example
```
// package.json
{
  "husky": {
    "hooks": {
      "pre-commit": "npm test",
      "pre-push": "npm test",
      "...": "..."
    }
  }
}
```